Makefile

```diff
@@ -10,53 +10,60 @@ endif
 
 .PHONY: install
 install: ## make install # Install roles dependencies
-	@ansible-galaxy install --roles-path vendor/ -r requirements.yml
```

Comment:
My challenge is that Ansible can't find the installed roles. I solved this problem by changing the roles-path option from "vendor" to "roles". I know it is posible to change the role_path since version 1.4 of Ansible (http://docs.ansible.com/ansible/latest/intro_configuration.html#roles-path) but I prefer the default behaviour rather the changed role_path. Another way to define the roles-path for the install target would be the use of a variable with a default value. e.g. make ROLES_PATH=roles install What do you think about this idea?